src/Team-Services-Bot.Api.UnitTests/Dialogs/ConnectDialogTests.cs

```diff
@@ -44,18 +42,22 @@ public ConnectDialogTests()
         [TestMethod]
         public async Task FirstTimeConnectionTest()
```

Comment:
The name of the test should be a little more descriptive so that if it fails I know from the name more about what has gone wrong. I often use a class called "GivenXYZ", with test methods inside called "WhenSomething_ThenThisThingHappens". I don't know what this test is going to test for from the name.